### PR TITLE
chore: Fix "FixedBy" for some E2E tests

### PR DESCRIPTION
**Pull request by jvdm**
Fixed By updated due to:

| name | version | vuln | os |  
|-|-|-|-|
| dotnet-runtime-6.0     | 6.0.6-1.el8_6.x86_64 | RHSA-2023:5710                | Red Hat |
| aspnetcore-runtime-6.0 | 6.0.6-1.el8_6.x86_64 | RHSA-2023:5710                | Red Hat |
| tomcat                 | 9.0.59               | CVE-2023-42795 CVE-2023-45648 | Alpine  |
| openssl | 3.0.2-0ubuntu1.12 | CVE-2023-2975 CVE-2023-3446 CVE-2023-3817 | Ubuntu |

This updates the E2E test.

### CODE REVIEW

**Comment by RTann** [comment]
thanks for handling this. looks like CI is still failing :(

**Review by daynewlee — APPROVED** [review]

File: e2etests/testcase_test.go [hunk 1]
```diff
@@ -3790,7 +3790,7 @@ Applications using RegexRequestMatcher with '.' in the regular expression are po
 				Name:          "openssl",
 				NamespaceName: "ubuntu:22.04",
 				VersionFormat: "dpkg",
-				Version:       "3.0.2-0ubuntu1.6",
```
Comment (RTann) [inline e2etests/testcase_test.go:3793]: This shouldn't change. This is the version of the package in the image. I think you meant to change "FixedBy"

**Pull request merged**